frontend/src/components/Spinner/Spinner.tsx

```diff
@@ -5,13 +5,25 @@ import S from './styles';
 
 type Props = {
   text: string;
+  component?: string;
 };
 
-const Spinner = ({ text }: Props) => (
-  <S.Wrapper className="loading-spinner">
-    <Spin indicator={<LoadingOutlined style={{ fontSize: '2.5rem' }} spin />} />
-    <p>{text}</p>
-  </S.Wrapper>
-);
+const Spinner = ({ text, component = '' }: Props) => {
```

Comment:
good idea, but maybe for reusability, we could instead make the `component` prop be `size`, that has a possible string type of `'small' | 'large'` (defaulting to large).